### PR TITLE
adding map elements

### PR DESCRIPTION
**Pull request by ceenell**
Working on a map of the focal sites for that page. This PR reflects intermediate steps to getting there. It adds:
- fetch targets to grab the IRB boundary and flowlines
- fetch targets with landcover and impervious surface cover through time (2001, 2004, 2006, 2008, 2011, 2016, 2019) for the basin using `FedData`. These targets take 20 min to fetch
- viz target with a basic map of the boundary, flowlines, and 3 focal sites
IRB_site_map

### CODE REVIEW

**Review by lindsayplatt — COMMENTED** [review]
Trying to build locally, but I can't get past this error in `get_nlcd_aoi()`:

```r
raster_aoi <- get_nlcd(template = aoi, label = aoi_label, year = nlcd_year , dataset = nlcd_dataset)
Error in (function (classes, fdef, mtable)  : 
  unable to find an inherited method for function ‘extent’ for signature ‘"sfc_POLYGON"’
```

I updated `sf` just in case and am using v1.0.8

**Comment by ceenell** [comment]
> Trying to build locally, but I can't get past this error in `get_nlcd_aoi()`:
> 
> ```r
> raster_aoi <- get_nlcd(template = aoi, label = aoi_label, year = nlcd_year , dataset = nlcd_dataset)
> Error in (function (classes, fdef, mtable)  : 
>   unable to find an inherited method for function ‘extent’ for signature ‘"sfc_POLYGON"’
> ```
> 
> I updated `sf` just in case and am using v1.0.8

hmm. I am using v1.0.7.

File: data-pipeline/1_fetch.R [hunk 1]
```diff
@@ -17,6 +34,37 @@ p1_targets_list <- list(
...
+  ),
+  tar_target(
+    p1_irb_nlcd,
+    get_nlcd_aoi(
+      aoi = p1_irb_sf,
+      aoi_label = 'IRB',
+      nlcd_dataset = 'landcover',
+      nlcd_year = years_nlcd,
+      file_name = sprintf('nlcd_%s.tif', years_nlcd),
+      out_folder = 'data-pipeline/1_fetch/out/nlcd'
```
[the remark this answers is not in the document]
Comment (ceenell) [reply data-pipeline/1_fetch.R:52]: done

**Review by lindsayplatt — APPROVED** [review]
Works now after installing `FedData` from GitHub!

**Pull request merged**